frontend/.eslintrc.js

```diff
@@ -128,7 +131,7 @@ module.exports = {
             // in `tsconfig.json`.
             "@typescript-eslint/no-unused-vars-experimental": "warn",
             "@typescript-eslint/semi": "warn",
-            "@typescript-eslint/indent": "warn",
+            "@typescript-eslint/indent": ["warn", ...indentOptions],
```

Comment:
Here, the options are not necessary because they are exactly the default for that rule, right?

Reply:
You're absolutely right.